### ISSUE TITLE
How to connect to a MongoDB SSL/TLS with adminMongo UI

### ISSUE THREAD

**Issue by kwill4026** [mrvautin/adminMongo#241]
I have my mongodb-replicaset configured with TLS/SSL support but I am unsure how to connect to it via the UI. Is there a specific connection string I should be using to connect to this replicaset which has SSL/TLS enabled?

**Comment by hackerunet** [comment]
was this solved???? how can I connect using tls and certificates?

**Comment by franklin432** [comment]
Nope. Check out bitnami/MongoDB chart and connect to that using the free MongoDB compass

**Comment by hackerunet** [comment]
Cannot use compass is too permisive i need a jailed environment without
client side installs and no file system access between hosts and clients
machines.

**Comment by hackerunet** [comment]
There is an easy "hack", I can add file read to the configuration file and works, but,. this client doesn't support aggregation pipelines. so it's useless as well.
